pyiron_atomistics/interactive/sxextoptint.py

```diff
@@ -164,13 +164,16 @@ def _write_input(
         selective_dynamics=False,
     ):
         with open(os.path.join(working_directory, "optim.sx"), "w") as f:
-            content = "main { ricQN { ric { maxDist = %f; withAngles; } maxSteps = %i; dEnergy = %f; dF = %f; maxStepLength = %f; softModeDamping = %f;}}" % (
-                maxDist,
-                ionic_steps,
-                ionic_energy_tolerance,
-                ionic_force_tolerance,
-                max_step_length,
-                soft_mode_damping,
+            content = (
+                "main { ricQN { ric { maxDist = %f; withAngles; } maxSteps = %i; dEnergy = %f; dF = %f; maxStepLength = %f; softModeDamping = %f;}}"
+                % (
+                    maxDist,
+                    ionic_steps,
+                    ionic_energy_tolerance,
+                    ionic_force_tolerance,
+                    max_step_length,
+                    soft_mode_damping,
+                )
             )
```

Comment:
I know it's not part of the change set, but this should really be constructed programmatically
```python
content = Group({'main': {'ricQN': {'ric': {...}, 'maxSteps': ionic_steps, ...}})
```